app/src/main/java/de/tum/in/tumcampusapp/tumonline/TUMOnlineRequest.java

```diff
@@ -105,7 +105,7 @@ public void cancelRequest(boolean mayInterruptIfRunning) {
             backgroundTask.cancel(mayInterruptIfRunning);
         }
     }
-
+    //added Utils.setSetting(c, Const.TUMO_DISABLED, false);
```

Comment:
????

Reply:
Oh sorry, that was a (bad) try to get rid of the token identification bug to solve one of the chatroom issues

Reply:
Ahhh, right. The diff catches all of those forgotten pieces of debugging attempts. 
(Also: theres a Debug button in your IDE for such cases :+1: )